dandi/tests/fixtures.py

```diff
@@ -107,6 +107,8 @@ def fixture():
 
 @pytest.fixture(scope="session")
 def local_docker():
+    if os.name != "posix":
+        pytest.skip("Docker images require Unix host")
```

Comment:
it doesn't work on OSX? (Darwin) I would have even assumed it might have worked on Windows.  Both have support for Docker.  What specific aspect is missing?

Reply:
`os.name` is `'posix'` on Darwin, so it will (and does) run fine on Macs.  (However, it'll be skipped on GitHub Actions' Mac environments, as they don't have Docker.)  As for Windows, `docker-compose` was failing on GitHub Actions' Windows environments with the message "image operating system "linux" cannot be used on this platform".  I believe we would have to build & upload separate images for Windows in order for the tests to run there.

Reply:
Thanks for the explanation!  Could you please also distill it as a comment in the code so anyone who runs into this skip would know why.